CreeDictionary/API/views.py

```diff
@@ -18,9 +15,7 @@ def click_in_text(request) -> HttpResponse:
     elif q == "":
         return HttpResponseBadRequest("query param q is an empty string")
 
-    results: List[SerializedSearchResult] = []
-    for result in simple_search(q, include_auto_definitions=False):
-        results.append(result.serialize(include_auto_definitions=False))
+    results = simple_search(q, include_auto_definitions=False)
```

Comment:
I just want to make sure we're not breaking the click-in-text API that's being used! So this should have tests ensuring the refactor did not, in fact, introduce existing changes.

Once that's handled, feel free to merge!